Cargo.toml

```
@@ -36,7 +36,7 @@ tiny-bip39 = "1.0.0"
 # This must be pinned to a version that lighthouse uses
 tree_hash = "0.5.2"
 tree_hash_derive = "0.5.2"
```

https://github.com/ChorusOne/lighthouse/blob/664a8fae9650ecba457fccbb561c5c0d6c270e3b/Cargo.toml#L216-L217

Revision 664a8f of Lighthouse uses newer versions. Can you please update them.
```suggestion
tree_hash = "0.8"
tree_hash_derive = "0.8"
```